### PR TITLE
Create Models for Profile,Category,Ads,Chat and register in Admin

### PR DESCRIPTION
**Pull request by DeepankumarTpDev**
Create Models for Profile, Category, Ads, Chat and registered in admin for Admin panel support. I have written unit test for all models.

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+
+
+class Profile(models.Model):
+    USER_TYPE_CHOICES = [
+        ('buyer', 'Buyer'),
+        ('seller', 'Seller'),
+    ]
```
Comment (shantanutomarr) [inline accounts/models.py]: I don't think a type is needed. Like on OLX, a user should be able to buy as well as sell stuff.

File: ads/models.py [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+    tags = TaggableManager()  # Using TaggableManager for tags
+    location = models.CharField(max_length=255, blank=False, null=False)
+    postal_code = models.CharField(max_length=20, blank=False, null=False)
+    contact_info = models.CharField(max_length=255, blank=False, null=False)
+    price = models.DecimalField(max_digits=10, decimal_places=2, blank=False, null=False)
+    show_contact_info = models.BooleanField(default=True)
+    event_start_date = models.DateTimeField(null=True, blank=True)
+    event_end_date = models.DateTimeField(null=True, blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```
Comment (shantanutomarr) [inline ads/models.py:39]: There is no user field in the ad ? Who posted this ad ?

File: accounts/models.py (hunk 1 above)
Comment (DeepankumarTpDev) [reply accounts/models.py]: ok sir I will resolve it.

File: ads/models.py (hunk 2 above)
Comment (DeepankumarTpDev) [reply ads/models.py:39]: i will add the user field sir.

**Review by shantanutomarr — APPROVED** [review]

**Pull request merged**